GameData/RealismOverhaul/Engine_Configs/NAA75_110_Config.cfg

```diff
@@ -220,7 +283,7 @@
 	TESTFLIGHT
 	{
 		name = A-7
-		ratedBurnTime = 155
+		ratedBurnTime = 143 // See: https://github.com/KSP-RO/RealismOverhaul/pull/2488#discussion_r676286211
```

Comment:
I ended up leaving it as A-7 in the configs, because I needed it to match the Part Browser for testing but could not change the part config name in Part Browser without manually editing files. Everything should match A-7 / A-7B in config, but you mentioned something about save game breaking so I can switch which one is actually listed as the default A-7. Would have to leave everything to do with Part Browser to you, then.

Reply:
Looking deeper at the entrycosts, then yes it's safe enough leaving that as A-7 and adding the hydyne variant as A-7B or A-7H or something with a PartBrowser ECM of just "HydyneFuel, A-7" and changing the A-7's Part Browser to just "A-6, 5000" -- then everything should just work.